redis/reply_test.go

```diff
@@ -225,6 +226,81 @@ func TestSlowLog(t *testing.T) {
 	}
 }
 
+func TestLatency(t *testing.T) {
+	c, err := dial()
+	require.NoError(t, err, "TestLatency failed during dial.")
+	defer c.Close()
+
+	resultStr, err := redis.Strings(c.Do("CONFIG", "GET", "latency-monitor-threshold"))
+	require.NoError(t, err, "TestLatency failed during CONFIG GET latency-monitor-threshold.")
```

Comment:
Messages should not include the test name, more below and see comment thread for more details.